code/src/App.js

```diff
@@ -1,12 +1,37 @@
 import React from 'react'
 import data from './data.json'
+import './app.css'
+import { Artist } from "./components/Artist";
+import { Album } from "./components/Album";
 
-console.log(data)
+const renderAlbum = album => {
+  const { id, images, name, external_urls } = album;
```

Comment:
Awesome that you're using destructuring! You could go one step further and remove a little more code by moving up to where you handle the function arguments:

```js
const renderAlbum = ({ id, images, name, external_urls }) => {
  // ...
}
```